vector/src/main/java/im/vector/app/features/roomprofile/RoomProfileFragment.kt

```diff
@@ -269,45 +245,9 @@ class RoomProfileFragment @Inject constructor(
 
     private fun onAvatarClicked(view: View, matrixItem: MatrixItem.RoomItem) = withState(roomProfileViewModel) {
         if (matrixItem.avatarUrl?.isNotEmpty() == true) {
-            val intent = BigImageViewerActivity.newIntent(requireContext(), matrixItem.getBestName(), matrixItem.avatarUrl!!, it.canChangeAvatar)
+            val intent = BigImageViewerActivity.newIntent(requireContext(), matrixItem.getBestName(), matrixItem.avatarUrl!!)
```

Comment:
Can we use `matrixItem.avatarUrl?.takeIf { it.isNotEmpty() }?.let { ... } `please?

Reply:
Done